src/Native/LibTorchSharp/THSTensor.cpp

```diff
@@ -3121,6 +3359,61 @@ Tensor THSInit_eye_(Tensor tensor)
     CATCH_TENSOR(torch::nn::init::eye_(*tensor))
 }
 
+Tensor THSTensor_bernoulli(const Tensor tensor, const double p, const Generator gen)
```

Comment:
Just to mention I'd been ordering the file end-to-end alphabetical, rather than by kind (distributions etc.)

I don't really mind, I just figured alphabetical was the only thing which didn't requrie a lot of thought :-)

Reply:
Yeah, but that had already started to break down a bit (my fault). I see value in having things that are related to each other close, but the real way to do that would be to break it into multiple .cpp files, not have one gigantic file. Maybe I should do that next...

Reply:
I have some changes that separates the C++ code into multiple files, but I'll put that in a different PR, since it makes it much harder to compare and review.